docs/source/usage/cli.rst

```diff
@@ -308,6 +308,24 @@ If ``console_enabled`` is ``true``, we can access the serial console using follo
 If ``console_enabled`` is ``false`` or ``console_info`` is ``None`` then
 the serial console is disabled. Note, there can only be one ipmi connection to the node, meaning only one user may access the console at a time.
 
+Rescue Mode
+-----------
+
+A node can be booted in rescue mode, potentially allowing for some form of remediation if some aspect of the node fails. Doing so puts the node
+on the rescue network temporarily to allow the rescue image to boot; then the node is returned to its original network configuration. If the node
+has an external floating IP attached, you can then access the node by running ``ssh rescue@<floating-ip>`` and using the password specified in
+the rescue command.
```

Comment:
can we make a note that the password specified should be a strong password?

Reply:
Seems very reasonable - done!